geoips_clavrx/plugins/modules/readers/clavrx_hdf4.py

```diff
@@ -175,3 +177,17 @@ def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=F
     # istat, outputs= read_cloudprops(fname, minlon, maxlon, minlat, maxlat)
     xarrays = read_cloudprops(fname, chans=chans, metadata_only=metadata_only)
     return {"DATA": xarrays, "METADATA": xarrays[[]]}
+
+
+def yeild_test_files():
```

Comment:
This should be updated to match the function names from geoips, right? `gen_test_files` rather than `yield_test_files`?

Also, should this include `gen_test_parameters`?

Reply:
@AJThorpe can you finish this PR up to use the setup in the geoips repo?